src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -29,6 +29,88 @@
 using namespace std;
 using namespace Microsoft::Console::Interactivity;
 
+HRESULT PseudoConsoleWindowAccessibilityProvider::RuntimeClassInitialize(HWND pseudoConsoleHwnd) noexcept

```

Comment:
If we're gonna keep this, we might as well keep it in its own file! make sure to add that file to `sources` as well.